### ISSUE TITLE
Unbroken Seal

### ISSUE THREAD

**Issue by LeahLuong** [CleverRaven/Cataclysm-DDA#55298]
### Describe the bug

This looks like a case that escaped the purview of #46520:

![Screenshot_20220211-102513](https://user-images.githubusercontent.com/91638992/153682652-d3da7e33-841c-4f1c-b3f8-b3f85d80993b.png)

I obtained this item by 'r'eloading an oil lamp.

### Steps to reproduce

1. Get the sealed lamp oil.
2. Put it in an oil lamp.
3. ???
4. PROFIT!

### Expected behavior

I expected that unsealing level would increase, because all other skills can increase in level.

### Versions and configuration

Android 0.F-3 Tiles English vanilla

### Additional context

I attempted to refill the bottle using a well & it gave me the following error & lost its seal:

![Screenshot_20220211-102553](https://user-images.githubusercontent.com/91638992/153682983-f19ac8e6-21ed-4769-bc16-e0ea5d45dbfc.png)

**Comment by robotic-gamer** [comment]
I also think I found a variation of this where some gallon jugs got like that from their contents rotting to nothingness.

**Comment by LeahLuong** [comment]
I don't believe all sealed beverages are shelf-stable. Rotten foodstuffs do disappear after a time. That's kind of a separate issue.

Here, we're concerned w/ being able to remove the contents of sealed containers on demand w/out breaking the seal. There, it should prob be the case that rotten beverages do not disappear from sealed containers.

**Comment by LeahLuong** [comment]
Strangely, 'r'eloading a gallon jug of clean water from a sealed plastic bottle broke the seal. Could the seal check be liquid-specific??
